src/forms/ThresholdsSelection.tsx

```diff
@@ -92,120 +91,134 @@ class ThresholdsSelectionInput extends Component<MyProps & InjectedIntlProps, My
+            <table>
+                <th
+                    colSpan={2}
+                    className={styles.Comparision}
+                >
+                    <FormattedMessage
+                        id="notifications_app.comparison"
+                        defaultMessage="Comparison"
+                    />
+                </th>
+                <tbody>
+                    <tr>
+                        <td>
                             <button
-                                className={
-                                    comparison === ">" ?
-                                        `${styles.SelectedButton}`
-                                        :
-                                        `${styles.SelectedButton} ${styles.UnselectedButton}`
-                                }
+                                className={comparison === ">" ? (
+                                    `${styles.SelectedButton}`
+                                ) : (
+                                    `${styles.SelectedButton} ${styles.UnselectedButton}`
+                                )}
                                 onClick={() => valueChanged({
-                                        comparison: ">",
-                                        thresholds: thresholds
-                                    })
-                                }
+                                    comparison: ">",
+                                    thresholds: thresholds
+                                })}
                             >
                                 <FormattedMessage
                                     id="notifications_app.higher_than"
                                     defaultMessage="higher than"
                                 />
                                 &nbsp;&gt;
                             </button>
-                            <label htmlFor="value">
-                                <FormattedMessage
-                                    id="notifications_app.threshold_value"
-                                    defaultMessage="Value"
-                                />
-                            </label>
-                            <div className={styles.ThresholdValues}>
-                                {thresholds.map((threshold, i) => <div key={i}>{comparison} {threshold.value}</div>)}
-                            </div>
-                            <input
-                                type="number"
-                                id="value"
-                                value={thresholdValue}
-                                className={formStyles.FormControlSmall}
-                                onChange={this.handleChangeThresholdValue}
-                            />
-                        </div>
-                        <div className={styles.ThresholdNameInput}>
+                        </td>
+                        <td>
                             <button
```

Comment:
These buttons do either not below in the table or otherwise belong in the header of the table.
In my opinion they should not be in the body.

Reply:
Moved to table header. To make them vertically align with other items in the column, I keep them in the table as well.